doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -92,16 +92,18 @@ It is not used directly by Matter applications, but it is used to generate the s
 The ZAP file can be edited using `ZCL Advanced Platform`_ (ZAP tool), a third-party tool that is a generic templating engine for applications and libraries based on Zigbee Cluster Library.
 This tool is provided with the Matter repository in the |NCS|.
 
+This guide uses the :ref:`ug_matter_gs_tools_matter_west_commands` to run the ZAP tool gui application and generate the output ZAP source files.
+To learn more about Matter west commands and how to use the ZAP tool, see the :ref:`ug_matter_gs_tools_matter_west_commands_zap_tool` user guide.
```

Comment:
```suggestion
This guide uses the :ref:`ug_matter_gs_tools_matter_west_commands_zap_tool` to install and run the ZAP tool GUI, and generate the data model's C++ source files.
```
conciseness suggestion to link directly to the relevant subsection + "ZAP source files" may not convey that the tool generates c++ sources.